lib/debugger/clients/wddebugger.js

```diff
@@ -103,6 +99,11 @@ WdDebugger.prototype.initRepl_ = function() {
 
   // We want the prompt to show up only after the controlflow text prints.
   this.dbgRepl.printControlFlow_(function() {
+    // Backward compatibility: node version 0.8.14 has a number of built in 
```

Comment:
I don't think we need to worry about versions <10 anymore, feel free to remove this.